src/libraries/Common/src/System/Collections/Generic/ReferenceEqualityComparer.cs

```diff
@@ -22,7 +22,7 @@ public bool Equals(T x, T y)
 
         public int GetHashCode(T obj)
         {
-            return RuntimeHelpers.GetHashCode(obj);
+            return RuntimeHelpers.GetHashCode(obj!);
```

Comment:
@Marusyk Why did you change this file? This was not showing up on the CI error and this is out of the scope of #2128. Can you please undo it?

Reply:
now CI failures with CS8604

Reply:
See https://github.com/dotnet/runtime/pull/31756#discussion_r375589070

Reply:
Ohh, just missed it. Thanks